packages/primitives/package.json

```diff
@@ -66,7 +65,7 @@
     "clean:build": "rm -rf dist abort-controller blob console crypto encoding events fetch streams structured-clone url",
     "clean:node": "rm -rf node_modules",
     "prebuild": "pnpm run clean:build",
-    "test": "jest"
+    "test": "#"
```

Comment:
nit: Perhaps you want to revert?

Reply:
I need to remove the `test` script altogether
it is now tested in `packages/integration-tests`, as the previous tests were depending on `jest-environment` and created a circular dependency loop, so `pnpm run test` failed without building in advance <picture data-single-emoji=":wow:" title=":wow:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/wow/8524bc5963b93848.gif" alt=":wow:" align="absmiddle"></picture>